lib/bank_accounts/checking_account.rb

```diff
@@ -0,0 +1,100 @@
+module BankAccounts
+  class CheckingAccount < Account
+    ##--------------------------------------------------------------------------
+    # CONSTANTS
+
+
+    # limits
+    CHECKS_LIMIT = 3
+    MINIMUM_BALANCE = -10
+
+    # fees
+    EXCESS_CHECKS_FEE = 2.00
+    WITHDRAWAL_FEE = 1.00
+
+
+    ##--------------------------------------------------------------------------
+    # INSTANCE METHODS
+    attr_reader :checks_used
+
+
+    def initialize(id, initial_balance)
+      super(id, initial_balance)
+
+      @checks_used = 0
+
+      return true
+    end
```

Comment:
`initialize` doesn't need to return a value (explained in a different comment cause I'm jumping around)